src/presentation/web/src/components/pageAdopt/adoptionStepCard/AdoptionSteps.tsx

```diff
@@ -0,0 +1,46 @@
+const stepsArray = [
```

Comment:
W sumie jest to tylko tablica więc wystarczyłoby rozszerzenie pliku ( ale nic nie szkodzi może tak zostać): .ts